modular_skyrat/code/modules/mob/living/carbon/human/species_types/synthetic/ipc.dm

```diff
@@ -28,6 +28,16 @@
 	languagewhitelist = list("Encoded Audio Language")
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/ipc
 
+/datum/species/ipc/spec_life(mob/living/carbon/human/H)
+	. = ..()
+	//This is stupid, performance intensive, and i want to shoot myself but it will shut people up
+	//about this minor fucking bug fuck you
+	if(H.stat == DEAD)
+		if((H.health > revivesbyhealreq) && !H.hellbound)
+			if((NOBLOOD in species_traits) || (H.blood_volume >= BLOOD_VOLUME_OKAY))
+				owner.revive(0)
+				owner.cure_husk(0) // If it has REVIVESBYHEALING, it probably can't be cloned. No husk cure.
```

Comment:
remove this, https://github.com/Skyrat-SS13/Skyrat13/pull/3595 fixes it

Reply:
it does not